### PR TITLE
updated resource instance example to latest tfv1.0

### PR DESCRIPTION
**Pull request by geethasathyamurthy**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: examples/ibm-resource-instance/variables.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 variable "service_name" {
-  default = "myservice"
+  default = "myservice18febgsm"
```
Comment (hkantare) [inline examples/ibm-resource-instance/variables.tf:2]: Can you remove this feb18

File: examples/ibm-resource-instance/README.md [hunk 2]
```diff
@@ -53,29 +47,32 @@ data "ibm_resource_instance" "test" {
...
 | Name | Description | Type | Required |
 |------|-------------|------|---------|
-| name | Name of the resource group. | `string` | yes |
 | service_name | The name of the service offering. | `string` | yes |
 | service_type | The type of the service offering. | `string` | yes |
 | plan| The name of the plan type supported by service.| `string` | yes |
 | location | Target location or environment to create the resource instance. | `string` | yes |
-
+| resource_group | Name of your resource group. | `string` | yes |
+{: caption="Resource input variables" caption-side="top"}
```
Comment (hkantare) [inline examples/ibm-resource-instance/README.md:77]: {: caption="Resource input variables" caption-side="top"} is not rendered proeprly

File: website/docs/d/satellite_location.html.markdown [hunk 3]
```diff
@@ -42,7 +42,7 @@ In addition to all argument reference list, you can access the following attribu
     * `zone`- The name of the zone
     * `host_labels`- Host Labels
 - `logging_account_id` - (String) The account ID for IBM Cloud Log Analysis with IBM Cloud Log Analysis log forwarding.
-- `managed_from` - (String) The IBM Cloud metro from which the Satellite location is managed. To list available multizone regions, run `ibmcloud ks locations`. such as `wdc04`, `wdc06`, or `lon04`.
+- `managed_from` - (String) The IBM Cloud regions that you can choose from to manage your Satellite location. To list available multizone regions, run `ibmcloud ks locations`. For more information, refer to [supported IBM Cloud locations](https://cloud.ibm.com/docs/satellite?topic=satellite-sat-regions).
```
Comment (hkantare) [inline website/docs/d/satellite_location.html.markdown:45]: why this changes are part of this pr

Comment (geethasathyamurthy) [reply website/docs/d/satellite_location.html.markdown:45]: proabably, I cloned from the master, and this edit is still in the progress to merge? Not sure, I am guessing Harini. Because this was in earlier PR.

File: examples/ibm-resource-instance/variables.tf (hunk 1 above)
Comment (geethasathyamurthy) [reply examples/ibm-resource-instance/variables.tf]: updated

File: examples/ibm-resource-instance/README.md (hunk 2 above)
Comment (geethasathyamurthy) [reply examples/ibm-resource-instance/README.md]: deleted the caption in the table.

**Review by geethasathyamurthy — COMMENTED** [review]
Updated examples folder, Harini

**Comment by hkantare** [comment]
fix conflict
